README.md

```diff
@@ -5,7 +5,7 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/muffin/obfuscate.svg?style=flat-square)](https://packagist.org/packages/muffin/obfuscate)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](LICENSE)
 
-CakePHP 3 support for ID obfuscation.
+CakePHP 3 support for ID obfuscation using HashIds, Optimus and Tiny.
```

Comment:
Replace `and` by `or` and if we are to mention them, maybe mention also or any custom strategy.